mapviz_plugins/include/mapviz_plugins/attitude_indicator_plugin.h

```diff
@@ -93,6 +93,7 @@ class AttitudeIndicatorPlugin : public mapviz::MapvizPlugin
 
   void Transform() {}
 
+  virtual void UpdateConfig(std::map<std::string, std::string>& params);
```

Comment:
virtual keyword is not needed here or the other plugins since they aren't intended to be sub-classed.  It doesn't hurt anything, but it is inconsistent with the other overload declarations.